cmake/FlowCommands.cmake

```diff
@@ -147,6 +147,42 @@ function(strip_debug_symbols target)
   add_dependencies(strip_targets strip_${target})
 endfunction()
 
+function(copy_headers)
```

Comment:
Usually FDB CMakefiles do not have any comments, but having a short explanation for routine motivation would help (especially future code maintenance).
Thoughts?

Reply:
Agreed, I will add a comment